js/index.js

```diff
@@ -77,7 +95,87 @@ function renderRain(rainPercentage) {
     element.innerText = percentage;
 }
 
+function renderWind(wind) {
+    const temp = wind.toFixed(2);
+    const element = document.getElementById("wind");
+
+    element.innerText = temp;
+}
+
+
 function convertKelvinToCelsius(tempInKelvin) {
     const diffrence = 273.15;
     return tempInKelvin - diffrence;
-}
\ No newline at end of file
+}
+
+const getWeekFromToday = (date) => {
+    const weekFromToday = [];
+    weekFromToday.push(...days.slice(date.getDay(), days.length),...days.slice(0, date.getDay()) );
```

Comment:
Chwilę zajęło mi zrozumienie jak to działa ale bardzo mi się podoba jak pomysłowo to zrobiłeś 👍